### PR TITLE
Remove broken downloads badge

### PR DESCRIPTION
**Pull request by hugovk**
See https://github.com/badges/shields/issues/716.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -7,7 +7,7 @@
 ==============================================================================
 pypiserver - minimal PyPI server for use with pip/easy_install
 ==============================================================================
-|pypi-ver| |travis-status| |dependencies| |downloads-count| |python-ver| \
+|pypi-ver| |travis-status| |dependencies| |python-ver| \
 |proj-license|
```
Comment (mplanchard) [inline README.rst]: Might as well pull `|proj-license|` up onto the same line so we don't need the backslash line continuation

Comment (hugovk) [reply README.rst]: Done!

**Comment by hugovk** [comment]
Stats are still available, now via BigQuery. https://github.com/ofek/pypinfo is a handy tool to access them.

For example, the stats in https://github.com/pypiserver/pypiserver/issues/192#issuecomment-349288254 are from there, and exclude downloads from mirrors (although mirror downloads are available too).

Here's updated numbers for May 2018:

| python_version | percent | download_count |
| -------------- | ------: | -------------: |
| 2.7            |  93.12% |          8,583 |
| 3.6            |   5.26% |            485 |
| 3.5            |   1.28% |            118 |
| 2.6            |   0.22% |             20 |
| 3.4            |   0.11% |             10 |
| 3.7            |   0.01% |              1 |
| Total          |         |          9,217 |

Now 2.6 is dropping and 3.2 has been replaced by 3.7 :)

**Comment by mplanchard** [comment]
I'm always surprised when I see how few folks have switched to 3. Only a year and a half left now!

**Comment by hugovk** [comment]
Yes, it varies a lot depending on the project. Some are very mostly 2.7, some are mostly 3.x.

pip

| python_version | percent | download_count |
| -------------- | ------: | -------------: |
| 2.7            |  91.63% |     20,557,971 |
| 3.6            |   3.62% |        811,150 |
| 3.5            |   2.62% |        588,043 |
| 3.4            |   1.42% |        318,064 |
| 2.6            |   0.37% |         82,816 |
| 3.3            |   0.29% |         65,304 |
| 3.7            |   0.04% |          8,353 |
| None           |   0.02% |          3,898 |
| 3.2            |   0.00% |            121 |
| 3.8            |   0.00% |             31 |
| Total          |         |     22,435,751 |

numpy

| python_version | percent | download_count |
| -------------- | ------: | -------------: |
| 2.7            |  48.19% |      1,349,449 |
| 3.5            |  25.71% |        720,002 |
| 3.6            |  22.33% |        625,337 |
| 3.4            |   3.53% |         98,777 |
| 3.7            |   0.18% |          5,115 |
| 3.3            |   0.04% |          1,033 |
| 2.6            |   0.03% |            730 |
| 3.2            |   0.00% |             36 |
| 3.8            |   0.00% |             16 |
| None           |   0.00% |             16 |
| Total          |         |      2,800,511 |

Pillow

 python_version | percent | download_count |
| -------------- | ------: | -------------: |
| 2.7            |  48.13% |        480,056 |
| 3.6            |  32.89% |        328,008 |
| 3.5            |  15.01% |        149,663 |
| 3.4            |   3.69% |         36,837 |
| 3.7            |   0.19% |          1,883 |
| 2.6            |   0.06% |            591 |
| 3.3            |   0.03% |            274 |
| 3.2            |   0.00% |             10 |
| None           |   0.00% |              9 |
| 3.8            |   0.00% |              2 |
| Total          |         |        997,333 |

tldextract

| python_version | percent | download_count |
| -------------- | ------: | -------------: |
| 2.7            |  65.11% |         76,370 |
| 3.6            |  14.82% |         17,386 |
| 3.5            |  14.82% |         17,378 |
| 3.4            |   5.16% |          6,050 |
| 3.7            |   0.07% |             84 |
| 2.6            |   0.02% |             23 |
| 3.3            |   0.00% |              1 |
| Total          |         |        117,292 |

CacheControl

| python_version | percent | download_count |
| -------------- | ------: | -------------: |
| 2.7            |  45.56% |         13,956 |
| 3.6            |  36.29% |         11,118 |
| 3.4            |   9.33% |          2,858 |
| 3.5            |   8.64% |          2,647 |
| 3.3            |   0.08% |             24 |
| 3.7            |   0.07% |             21 |
| 2.6            |   0.03% |              8 |
| 3.8            |   0.00% |              1 |
| Total          |         |         30,633 |

Have a look at http://python3statement.org/ and http://python3statement.org/practicalities/ to see how some are planning on deprecating and then dropping 2.7.

**Pull request merged**